website/docs/reference/halving.md

```diff
@@ -0,0 +1,39 @@
+---
+id: halving
+title: Halving
+---
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+The base issuance of CKBytes are rewarded to miners for securing the network, and the reward halves approximately every **4 years** until all the base issuance tokens (**33.6 billion**) are mined out.
+
+## How CKByte-Halving works?
+
+In order to make the halving plan work as expected, a concept of time called **epoch** was introduced.
+
+>An epoch is a period of time for a set of blocks. 
+>
+>In Nervos, the PoW difficulty changes on a new epoch. All the blocks in the same epoch share the same difficulty target. The difficulty adjustment algorithm aims to stabilize the orphan block rate at 2.5% and the epoch duration at 4 hours.
+
+Epochs per halving is `4 * 365 * (24 / 4)` = `8760`, and the Nth halving of CKBytes firstly occurs on epoch: `the_Nth_halving_epoch = 8760 * N `.
+
+So, The CKByte halving event occurs on the specified epoch, e.g. 8760, 17520. 
+
+## When will CKByte be halved?
+The following are the next 10 halving windows of CKBytes, and the corresponding base issuance rewards.
+
+
+|Event          |Date                  |Epoch number      |Epoch reward  |Daily reward  |Total new CKB between events  |
+|---------------|----------------------|------------------|-------------:|--------------:|----------------------------:|
+|Nervos launches|16 November 2016      |0 (genesis epoch) |1,917,808 CKB |11,506,849 CKB |16,800,000,000 CKB           |
```

Comment:
It would have been great if ckb had launched in 2016, but it's in 2019.

Reply:
https://github.com/yixyxiu/docs-new/commit/513ea80d05d1998aeca0b693e8c3f06ff603cf4f